corehq/apps/es/domains.py

```diff
@@ -79,7 +79,9 @@ def last_modified(gt=None, gte=None, lt=None, lte=None):
 
 
 def in_domains(domains):
-    return filters.term('name', list(domains))
+    if type(domains) is not list:
+        domains = [domains]
+    return filters.term('name', domains)
```

Comment:
Good catch.  [`filters.term`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/filters.py#L24-L34) does this anyways, so I think the best fix here would just be to drop the `list()`.  The convention I've been using for things that simply match against a field is to name the filter after the field, and pass it straight to `filters.term`, which accepts a list, a tuple, or a singleton.

Also, for future reference, generally `isinstance(domains, list)` is the way to do this kind of type checking, as it'll catch subclasses as well.  For instance, `string`s and `unicode`s are both instances of `basestring`.